lib/config.yml.hou

```diff
@@ -0,0 +1,212 @@
+# react-router config, for URL routing (not to be confused with multimodal/OTP routing)
```

Comment:
We should remove this file now that it is possible to send in the env.OTP_CONFIG to webpack.